internal/restapi/v1/integrations/tests/post_test.go

```diff
@@ -0,0 +1,120 @@
+package tests
```

Comment:
After we make [this](https://github.com/shinobistack/gokakashi/pull/43#pullrequestreview-2518539333) change, we can either have the package name for tests to be 
- `integrations `: if you would like to write tests for both unexported and exported methods of a package.
- `integrations_test`: if you would like to write tests only for exported methods of a package.

Reply:
Picking the `integrations_test`. Since the package has simple logic and focusing on end-user behaviour.